Week1/homework/js-exercises/logNumber.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+let numberX;
+console.log(the value of x undefined. Because there is no value defined);
```

Comment:
Make sure that you wrap this string in the literal expressions to create a string `"`
If you don't tell the interpreter that its a string, they will be handled as variables. And in this case, all those variables are undefined, so it will give us a syntax error. 

Just add 
```suggestion
console.log("the value of x undefined. Because there is no value defined");
```